tests/admin/test_controller.py

```diff
@@ -1459,21 +1463,35 @@ def test_libraries_post_edit(self):
         eq_(library.short_name, "nypl")
 
         # The library-wide settings were updated.
-        eq_("20", ConfigurationSetting.for_library(Configuration.FEATURED_LANE_SIZE, library).value)
-        eq_("0.9", ConfigurationSetting.for_library(Configuration.MINIMUM_FEATURED_QUALITY, library).value)
+        def val(x):
+            return ConfigurationSetting.for_library(x, library).value 
+        eq_("https://library.library/", val(Configuration.WEBSITE_URL))
+        eq_("email@example.com", val(Configuration.DEFAULT_NOTIFICATION_EMAIL_ADDRESS))
+        eq_("20", val(Configuration.FEATURED_LANE_SIZE))
+        eq_("0.9", val(Configuration.MINIMUM_FEATURED_QUALITY))
         eq_(FacetConstants.ORDER_AUTHOR,
-            ConfigurationSetting.for_library(
-                Configuration.DEFAULT_FACET_KEY_PREFIX + FacetConstants.ORDER_FACET_GROUP_NAME,
-                library).value)
+            val(Configuration.DEFAULT_FACET_KEY_PREFIX + FacetConstants.ORDER_FACET_GROUP_NAME)
+        )
         eq_(json.dumps([FacetConstants.ORDER_AUTHOR, FacetConstants.ORDER_RANDOM]),
-            ConfigurationSetting.for_library(
-                Configuration.ENABLED_FACETS_KEY_PREFIX + FacetConstants.ORDER_FACET_GROUP_NAME,
-                library).value)
+            val(Configuration.ENABLED_FACETS_KEY_PREFIX + FacetConstants.ORDER_FACET_GROUP_NAME)
+        )
 
         # The library-wide logo was not updated and has been left alone.
         eq_("A tiny image", 
             ConfigurationSetting.for_library(Configuration.LOGO, library).value
         )
+
+    def test_libraries_post_errors(self):
```

Comment:
There was already a test_libraries_post_errors above the previous test, you might want to add on to that.